waiter/config-full.edn

```diff
@@ -263,6 +263,45 @@
                                ;; The URL for your Marathon HTTP API:
                                :url "http://marathon.example.com:8080"}
 
+                    ;; :kind :kubernetes uses Kubernetes (https://kubernetes.io/) for scheduling Waiter services and instances:
+                    ;:kind :kubernetes
+                    :kubernetes {
+                                 ;; Configuration for obtaining the credentials to authenticate with the Kubernetes API server.
+                                 ;; If authentication isn't required (e.g., using kubectl proxy), then this can be omitted.
+                                 :authentication {;; Number of minutes to wait between attempts to refresh the credentials.
+                                                  ;; If omitted, then the :refresh-fn value is only called once at Waiter startup.
+                                                  :refresh-delay-minutes 15
+
+                                                  ;; Nullary function which can be called periodically (or just once if :refresh-delay-minutes is omitted)
+                                                  ;; to obtain an Authorization string for use when authenticating with the Kubernetes API server.
+                                                  ;; The function must return a string that can be used as the value for the Authorization HTTP header.
+                                                  :refresh-fn waiter.scheduler.kubernetes/mock-auth-refresh}
```

Comment:
Can we point to an actual implementation instead of having a mock implementation in the main source code?

Reply:
Changed this mock implementation to a less contrived sample:
```clojure
(defn authorization-from-environment []
  "Sample implementation of the authentication string refresh function.
   Returns a string to be used as the value for the Authorization HTTP header,
   reading the string from the WAITER_K8S_AUTH_STRING environment variable."
  (log/info "Called waiter.scheduler.kubernetes/authorization-from-environment")
  (System/getenv "WAITER_K8S_AUTH_STRING"))
```

Reply:
Done.